specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -381,6 +381,48 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{serverName}/listGatewayStatus": {
```

Comment:
Why do you need a  separate POST API to get the gateway status? Cant it be obtained by GET /servers/{servername}?